src/styles/main.scss

```
@@ -1,3 +1,74 @@
+$calendar-block-color: #eee;
+$border-color: #000;
+
+$gap: 1px;
+$border: 1px;
+$your-block-size: 100px;
+
+@function calendarSize() {
+  @return 7 * $your-block-size + $gap * 9;
+}
+
+@function block-size() {
+  @return $your-block-size - ($border * 1.6);
+}
```

why do we need to use functions here, if you don't using any argument, conditions etc.
```suggestion
$calendar-size: 7 * $your-block-size + $gap * 9;
$block-size: $your-block-size - ($border * 1.6)
```